accessibility-checker.php

```diff
@@ -421,39 +458,25 @@ function edac_summary( $post_id ) {
 
 	$summary['content_grade'] = 0;
 	if ( class_exists( 'DaveChild\TextStatistics\TextStatistics' ) ) {
-		$text_statistics          = new DaveChild\TextStatistics\TextStatistics();
-		$summary['content_grade'] = floor( $text_statistics->fleschKincaidGradeLevel( $content ) );
+		$summary['content_grade'] = floor(
+			( new DaveChild\TextStatistics\TextStatistics() )->fleschKincaidGradeLevel( $content )
+		);
 	}
 
-	$summary['readability'] = ( 0 === $summary['content_grade'] ) ? 'N/A' : edac_ordinal( $summary['content_grade'] );
+	$summary['readability'] = 0 === $summary['content_grade']
+		? 'N/A'
+		: edac_ordinal( $summary['content_grade'] );
 
 	// simplified summary.
-	$summary['simplified_summary'] = get_post_meta( $post_id, '_edac_simplified_summary', true ) ? true : false;
+	$summary['simplified_summary'] = (bool) ( get_post_meta( $post_id, '_edac_simplified_summary', true ) );
 
 	// save summary data as post meta.
-	if ( ! add_post_meta( $post_id, '_edac_summary', $summary, true ) ) {
-		update_post_meta( $post_id, '_edac_summary', $summary );
-	}
-
-	if ( ! add_post_meta( $post_id, '_edac_summary_passed_tests', $summary['passed_tests'], true ) ) {
-		update_post_meta( $post_id, '_edac_summary_passed_tests', $summary['passed_tests'] );
-	}
-
-	if ( ! add_post_meta( $post_id, '_edac_summary_errors', $summary['errors'], true ) ) {
-		update_post_meta( $post_id, '_edac_summary_errors', $summary['errors'] );
-	}
-
-	if ( ! add_post_meta( $post_id, '_edac_summary_warnings', $summary['warnings'], true ) ) {
-		update_post_meta( $post_id, '_edac_summary_warnings', $summary['warnings'] );
-	}
-
-	if ( ! add_post_meta( $post_id, '_edac_summary_ignored', $summary['ignored'], true ) ) {
-		update_post_meta( $post_id, '_edac_summary_ignored', $summary['ignored'] );
-	}
-
-	if ( ! add_post_meta( $post_id, '_edac_summary_contrast_errors', $summary['contrast_errors'], true ) ) {
-		update_post_meta( $post_id, '_edac_summary_contrast_errors', $summary['contrast_errors'] );
-	}
+	update_post_meta( $post_id, '_edac_summary', $summary );
+	update_post_meta( $post_id, '_edac_summary_passed_tests', $summary['passed_tests'] );
+	update_post_meta( $post_id, '_edac_summary_errors', $summary['errors'] );
+	update_post_meta( $post_id, '_edac_summary_warnings', $summary['warnings'] );
+	update_post_meta( $post_id, '_edac_summary_ignored', $summary['ignored'] );
+	update_post_meta( $post_id, '_edac_summary_contrast_errors', $summary['contrast_errors'] );
```

Comment:
`update_post_meta` calls `update_metadata`.
[Documentation for the `update_metadata` function](https://developer.wordpress.org/reference/functions/update_metadata/) mentions this:
> If no value already exists for the specified object ID and metadata key, the metadata will be added.

With that in mind, calling `add_post_meta` before `update_post_meta` is unnecessary.

The resulting code is cleaner, faster, and easier to digest.